src/Fixer/CastNotation/ModernizeTypesCastingFixer.php

```diff
@@ -37,7 +37,8 @@ public function getDefinition()
     $a = doubleval($b);
     $a = strval ($b);
     $a = boolval($b);
-'),
+'
+            ),
```

Comment:
The issue here is that both the array and the arguments list must be fixed. But indeed, the fixer you are working on should only care about arguments list, so please fix this manually instead.

Implementing a fixer for the array would be a nice-to-have though.